crates/file_formats/dat/Cargo.toml

```diff
@@ -9,7 +9,7 @@ keywords = ["dat", "aoe", "age", "empires", "genie", "engine"]
 authors = ["Kevin Fuller <angered.ghandi@gmail.com>"]
 
 [dependencies]
-quick-error = "1.1"
+error-chain = "0.1.12"

```

Comment:
error-chain 0.2 is out now, and I think that's the version the other crates are using.